assets/stylesheets/bookingsync/_sweetalert.scss

```diff
@@ -13,5 +18,27 @@
 }
 
 .swal2-title {
+  color: $headings-color !important;
+  font-weight: 400 !important;
+  font-size: 22px !important;
+}
+
+.swal2-content {
+  font-size: 16px !important;
   font-weight: 400 !important;
 }
+
+.swal2-title, .swal2-content {
+  text-align: left !important;
+}
+
+.swal2-styled {
```

Comment:
Maybe in the end it would be good to use `setDefaults()` for `swal` as we can overwrite the buttons style in there like this  `swal.setDefauls({ buttonsStyling: false, confirmButtonClass: 'btn btn-primary', cancelButtonClass: 'btn btn-secondary });` we can also pass the `reverseButtons: true` option to defaults so don't need to overwrite that with CSS.